### PR TITLE
Update presence-admins.md

### PR DESCRIPTION
**Pull request by RuiPereiraTabares**
made changes based on current design

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/presence-admins.md [hunk 1]
```diff
@@ -46,7 +46,8 @@ Presence is part of a user's profile in Microsoft Teams (and throughout Microsof
 || ![Open gray circle, indicates status unknown](media/Presence_Unknown.png) Status unknown|
 || ![Purple circle with arrow, indicates Out of office](media/Presence_OOF.png) Out of Office. Out of Office is used when an automatic reply is set. (Available in Outlook only.) |
 |||
-
+ > [!Note]
```
Comment (JohanFreelancer9) [inline Teams/presence-admins.md:49]:
```suggestion
> [!NOTE]
```

File: Teams/presence-admins.md [hunk 2]
```diff
@@ -46,7 +46,8 @@ Presence is part of a user's profile in Microsoft Teams (and throughout Microsof
...
+ > For users that there mailbox is hosted onprem it is expected to have presence delays with a maximun of an hour.
```
Comment (JohanFreelancer9) [inline Teams/presence-admins.md:50]:
```suggestion
 > Users whose mailboxes are hosted on-premises can expect to experience presence delays of a maximum of one hour.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@msdmaguire PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**